res2df/gruptree.py

```diff
@@ -392,7 +392,9 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         parser (argparse.ArgumentParser or argparse.subparser): parser
             to fill with arguments
     """
-    parser.add_argument("DATAFILE", help="Name of Eclipse DATA file.")
+    parser.add_argument(
+        "DATAFILE", help="Name of the .DATA input file for the reservoir simulator"
```

Comment:
Just use this help string wherever there is an argument named "DATAFILE" for consistency.